recipes/sortmerna/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sortmerna" %}
+{% set version = "4.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6b62def30704ea956e1de060b602f9bebc8f2ba68107c59329b332500997b1d2
+
+build:
+  number: 0
+  run_exports: []
+  #skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Needs `stdlib`:

```suggestion
    - {{ compiler('c') }}
    - {{ stdlib('c') }}
```

Reply:
Adding `{{ stdlib('c') }}` throws
```
ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("c_linux-64")}
Encountered problems while solving:
  - nothing provides requested c_linux-64

Could not solve for environment specs
The following package could not be installed
└─ c_linux-64 does not exist (perhaps a typo or a missing channel).
```
works fine without it